apps/estuary/lib/estuary/services/event_retrieval_service.ex

```diff
@@ -4,18 +4,30 @@ defmodule Estuary.Services.EventRetrievalService do
   """
   alias Estuary.Query.Select
   alias Estuary.Datasets.DatasetSchema
+  alias Estuary.Query.Helper.PrestigeHelper
 
   def get_all() do
-    make_select_table_schema()
-    |> Select.select_table()
+    make_select_table_sub_schema()
+    |> Select.create_select_statement()
+    |> make_select_table_schema
+    |> Select.create_select_statement()
+    |> PrestigeHelper.execute_query()
   end
```

Comment:
It's not clear to me what's happening here.  You're piping one create_select_statement into another... is there a way we can do this that makes it a little clearer what the final query is?